docs/source/tutorial/multi_gpu_vanilla.rst

```diff
@@ -1,5 +1,7 @@
 Multi-GPU Training in Pure PyTorch
 ==================================
+Note that this method is deprecated in liue of CuGraph, the recommended multi-gpu method. See `CuGraph Examples Repo <https://github.com/rapidsai/cugraph-gnn/tree/main/python/cugraph-pyg/cugraph_pyg/examples>`_.
+
```

Comment:
I don't think this should be deprecated because it's still a valid way and is supported by PyTorch naitively.